packages/wasi-libc/build.sh

```diff
@@ -55,12 +55,6 @@ termux_step_host_build() {
 		-j ${TERMUX_PKG_MAKE_PROCESSES} \
 		install
 
-	mkdir -p "${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}"
-	mv -v "${TERMUX_PREFIX}/VERSION" "${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}"
-	echo "INFO: ${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}/VERSION"
-	cat "${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}/VERSION"
-	echo
```

Comment:
So, where did the docs go then?

Reply:
It goes to nothing, the CMake doesn't install any related `VERSION` file into the root of `TERMUX_PREFIX`.

Reply:
What?

Reply:
The reason I removed that lines because it failed to move `VERSION` in `$TERMUX_PREFIX`, which is a non-existant file. Your question is confusing, but I hope this one answers what's in your mind.

Reply:
It's ok. VERSION file is installed by cmake correctly to `/data/data/com.termux/files/usr/share/wasi-sysroot/VERSION` now. No need for manual mv. https://github.com/WebAssembly/wasi-sdk/commit/b416e2beb40beecc05de33e300203843f62b3263